src/generate/apiCollection.test.ts

```diff
@@ -23,19 +23,17 @@ describe("Test ApiGroup class init", () => {
     const collection = await ApiCollection.init({
       "Group One": [validRamlFile]
     });
-    expect(collection.get("Group One").apis).to.not.be.empty;
-    expect(collection.get("Group One").apis[0].model).to.not.be.empty;
-    expect(collection.get("Group One").apis[0].name.original).to.equal(
-      "Shop API"
-    );
-    expect(collection.get("Group One").apis[0].path).to.be.equal(validRamlFile);
+    expect(collection.get("Group One")).to.not.be.empty;
+    expect(collection.get("Group One")[0].model).to.not.be.empty;
+    expect(collection.get("Group One")[0].name.original).to.equal("Shop API");
+    expect(collection.get("Group One")[0].path).to.be.equal(validRamlFile);
```

Comment:
```suggestion
    expect(collection.get("Group One")[0].path).to.equal(validRamlFile);
```
Also in a few other places...

Reply:
Also good catches